code/modules/mob/living/silicon/robot/drone/drone_say.dm

```diff
@@ -30,6 +34,8 @@
 				D << "<b>[src]</b> transmits, \"[message]\""
 
 		for (var/mob/M in player_list)
+			if (isnull(M) || isnull(M.client))
```

Comment:
Boot it from player_list as well. `null`s shouldn't be there in the first place. Though I think this was fixed in dev.